phoenix-core/src/main/java/org/apache/phoenix/util/PhoenixRuntime.java

```diff
@@ -221,6 +224,14 @@
             REQUEST_METRIC_ATTRIB,
             };
 
+    /**
+     * Constants to help determine what to bootstrap the HBase connection with.
+     */
+    public final static String BOOTSTRAP_ZK = "zk";
+    public final static String BOOTSTRAP_HRPC = "hrpc";
+    public final static String BOOTSTRAP_HRPC_DEFAULT_HMASTER_PORT = "60010";
```

Comment:
According to the https://hbase.apache.org/book.html, the default port is 16010 now.

However, I think that just using the **hbase.master.info.port** property value, and letting HBase figure out the default port would be the best option.

I think that the current code won't pick up the default port from hbase-site.xml if you specify the masters, but not the ports.